source/release/4.2.0/index.html.md

```diff
@@ -130,7 +130,7 @@ packages from other repos.
  - [BZ 1391859](https://bugzilla.redhat.com/1391859) <b>[RFE] When creating thick allocated disks on file-based storage, use '-o preallocation=falloc' instead of 'dd' with zeros.</b><br>Feature: Time required to create thick allocated disks on file-based storage was greatly improved.<br><br>Reason: Old approach was to write zeroes to fill the file using dd command. Now we use posix_fallocate(3) to ensures that disk space is allocated.<br><br>Result: Thick allocated volumes are created much faster now. In extreme case 1gb volume creation time reduced from 61 seconds to less then a second.
  - [BZ 1228543](https://bugzilla.redhat.com/1228543) <b>[RFE] hot-unplug memory</b><br>Memory hot-unplug feature is documented at<br>https://www.ovirt.org/develop/release-management/features/hot-unplug-memory/<br><br>When documenting the feature, please note that memory hot-unplug functionality places certain requirements on memory hot-plug. Those requirements should be mentioned not only in the memory hot-unplug documentation but also in the memory hot-plug documentation.
  - [BZ 1334982](https://bugzilla.redhat.com/1334982) <b>[RFE] Gracefully shutdown Virtual Machines on Host reboot/shutdown.</b><br>Feature: <br>Gracefully shutdown the running VMs on host shutdown.<br><br>Reason: <br>Without this feature the running VMs are killed by the systemd process. Killed VMs are not given any chance to finish proper shutdown, thus resulting in undefined states.<br>Undefined state results in undesirable scenarios, especially for VMs running databases, eg Oracle, SAP ...<br><br>Result: <br>The graceful shutdown takes care of running VMs by holding the systemd and telling VMs to shutdown properly. After VMs are shutdown, the systemd is allowed to take over control and continue shutdown.<br>The VDSM is shutdown after the graceful shutdown of VMs.<br>Before that VDSM passes information to the engine and waits for for (default, can be changed in vdsm.conf) 5s for the engine to ACK VMs shutdown.
- - [BZ 1168327](https://bugzilla.redhat.com/1168327) <b>Live Merge: optimize internal volume size</b><br>Feature: <br>Reduce (base) volume size to optimal after live merge.<br><br>Reason: <br>During live merge, the base volume is extended to accommodate merging the top volume into it.<br><br>Result: <br>A new verb, ReduceVolue, is now called at the of live merge to reduce (base) volume size to optimal.
+ - [BZ 1168327](https://bugzilla.redhat.com/1168327) <b>Live Merge: optimize internal volume size</b><br>Feature: <br>Reduce (base) volume size to optimal after live merge.<br><br>Reason: <br>During live merge, the base volume is extended to accommodate merging the top volume into it.<br><br>Result: <br>A new verb, reduceVolume, is now called at the of live merge to reduce (base) volume size to optimal.
```

Comment:
Looks like doc-text in https://bugzilla.redhat.com/show_bug.cgi?id=1168327 completely changed. Now is: "After live merge, the base volume is reduced to its optimal size." This is what will be written here next time I'll regenerate the release notes.

Reply:
The new content has been already reviewed by doc team so if "After live merge, the base volume is reduced to its optimal size." is not enough, please comment in the bug.